tensorflow/contrib/rnn/python/kernel_tests/rnn_cell_test.py

```diff
@@ -906,6 +906,56 @@ def testHighwayWrapper(self):
       # States are left untouched
       self.assertAllClose(res[2], res[3])
 
+  def testGLSTMCell(self):
+    #First test that GLSTM matches LSTM when number_of_groups = 1
+    batch_size = 2
+    num_units = 4
+    number_of_groups = 1
+
+    with self.test_session() as sess:
+      with variable_scope.variable_scope(
+          "root1", initializer=init_ops.constant_initializer(0.5)):
+        x = array_ops.ones([batch_size, num_units])
+        gcell = rnn_cell.GLSTMCell(num_units=num_units, number_of_groups=number_of_groups)
```

Comment:
Does number_of_groups== 1 reduce to LSTMCell?  If so, can you add a comment here about that?